fix: ADR 252 TPRv3 (#271)
decentraland/adr: content/ADR-252-third-party-registry-v3.md

In such cases, users can designate a third party as programmatic by setting the `_areProgrammatic` variable in the `addThirdParties` function as `true`.

Programmatic third parties are charged a fixed rate based on the `programmaticBasePurchasedSlots` variable. For example, if this variable is set to 20, a programmatic TP will be billed as if it had 20 item slots. However, there is no limit to the number of item slots a programmatic TP can actually have.

Given that programmatic Third Parties are reviewed by curators before being made available on the platform, users cannot exploit the different pricing models for programmatic and non-programmatic Third Parties. For instance, a user attempting to create a Third Party with 30 manually created items and marking it as programmatic to pay less would fail to comply with the committee's rules and would not be approved.

---

> The `buyItemSlots` function has been updated to allow programmatic Third Party managers to add an unlimited number of item slots for free if the initial amount defined was insufficient.

Reviewer: ```suggestion
The `buyItemSlots` function has been updated to allow programmatic Third Party managers to add more item slots for free if the initial amount defined was insufficient.
```
